cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/templates/etl/api/Transform.java

```diff
@@ -17,7 +17,9 @@
 package co.cask.cdap.templates.etl.api;
 
 import co.cask.cdap.api.ProgramLifecycle;
+import com.google.common.reflect.TypeToken;
```

Comment:
i guess guava is already a dependency?  Would be good to avoid putting it in api if possible but I guess we can revisit.